elements/pfe-navigation/src/pfe-navigation--lightdom.scss

```
@@ -555,7 +555,7 @@ pfe-navigation-item.expanded pfe-navigation-item[parent_hidden]:not(.expanded) [
 
 pfe-navigation-main [slot="trigger"],
 pfe-navigation-main [slot="trigger"] > a {
-    font-size: $trigger--FontSize--fallback;
+    font-size: #{pfe-local($region: trigger, $cssvar: FontSize, $fallback: $trigger--FontSize--fallback)};
```

```suggestion
    font-size: pfe-local(FontSize, $region: trigger);
```
No interpolation or fallback override necessary with the latest updates we've made to the Sass `pfe-local` function! 🥳